src/v8.zig

```diff
@@ -2299,6 +2299,8 @@ pub const Inspector = struct {
 
     rnd: RndGen = RndGen.init(0),
 
+    context: ?*const C_Context = null,
```

Comment:
Maybe we can rename this field as `ctx_handle` or something like that to be more clear about the fact that this is the C/C++ handle and not the zig Context wrapper.